.dotnet/src/Custom/Assistants/MessageCreationOptions.cs

```diff
@@ -29,10 +29,10 @@ public partial class MessageCreationOptions
     /// </summary>
     public MessageCreationOptions()
     : this(
-        MessageRole.User,
-        new ChangeTrackingList<MessageContent>(),
         new ChangeTrackingList<MessageCreationAttachment>(),
         new ChangeTrackingDictionary<string, string>(),
+        MessageRole.User,
+        new ChangeTrackingList<MessageContent>(),
         new ChangeTrackingDictionary<string, BinaryData>())
```

Comment:
Why did the parameter order change here?

Reply:
The ordering for the full constructor is not the same in MGC, but this is internal so it doesn't impact the API.